internal/providers/terraform/aws/ebs_snapshot_copy.go

```diff
@@ -1,43 +1,31 @@
 package aws
 
 import (
+	"github.com/infracost/infracost/internal/resources/aws"
 	"github.com/infracost/infracost/internal/schema"
-
-	"github.com/shopspring/decimal"
 )
 
-func GetEBSSnapshotCopyRegistryItem() *schema.RegistryItem {
+func getEBSSnapshotCopyRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
 		Name:  "aws_ebs_snapshot_copy",
-		RFunc: NewEBSSnapshotCopy,
+		RFunc: NewEbsSnapshotCopy,
 		ReferenceAttributes: []string{
 			"volume_id",
 			"source_snapshot_id",
 		},
 	}
 }
-
-func NewEBSSnapshotCopy(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
-	region := d.Get("region").String()
-
-	gbVal := decimal.NewFromInt(int64(defaultVolumeSize))
-
+func NewEbsSnapshotCopy(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	r := &aws.EbsSnapshotCopy{Address: d.Address, Region: d.Get("region").String()}
```

Comment:
Do we need a separate `EBSSnapshotCopy` resource here or could we just use the `EBSSnapshot` resource?

Reply:
Nice catch, used the EBSSnapshot

Reply:
The outputs need to be different so I will return back the copy.